File: ad-hoc/javascript/3474.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.on('line', line => {
    const [valorFigurinha, numIntegrantes] = line.split(' ').map(Number);
    const valorPorIntegrante = valorFigurinha / numIntegrantes;
    console.log(valorPorIntegrante.toFixed(2));
});

File: ad-hoc/c/3474.c
#include <stdio.h>
 
int main() {
    double valorFigurinha;
    int numIntegrantes;

    while (scanf("%lf %d", &valorFigurinha, &numIntegrantes) == 2) {
        double valorPorIntegrante = valorFigurinha / numIntegrantes;
        printf("%.2lf\n", valorPorIntegrante);
    }
 
    return 0;
}
